### DISCUSSION TITLE
Running Samples in Visual Studio 2015

### DISCUSSION THREAD

**Discussion by jslezosky** [biojppm/rapidyaml#227] [Q&A] [answered]
I can run rapidyaml/samples/singleheader/run.sh in powershell on my windows development machine and it runs to SUCCESS!
I can't figure out how to build and run the singleheader example in Visual Studio 2015.

The README.MD says "ryml is written in C++11, and compiles cleanly with: Visual Studio 2015 and later" but I can't even get the solution to load. 

Can someone explain the steps required to load, build and run the Samples in Visual Studio 2015?

**Comment by biojppm** [comment, marked as answer]
Just look inside the `run.sh` script; it is there to show what commands to run. To pick specifically vs2015 it would require something like this (assuming a recent cmake is installed):
```console
cd rapidyaml/samples/singleheader
cmake -G "Visual Studio 14 2015" -A x64 -S . -B build
cmake --build build --config Release --target run
```

**Reply by biojppm** [reply]
BTW, sometime ago Github retired the vs2015 runners from its CI. The workflows testing vs2015 had to be disabled, and as a consequence, I can no longer guarantee that rapidyaml is compiled cleanly under vs2015 (the README needs to be updated to state this). For now, feel free to open an issue if anything is off.

**Reply by biojppm** [reply]
As an ironic coincidence, github just retired the vs2017 runners as well.

**Reply by jslezosky** [reply]
Thank You. The above instructions worked and allowed me to build and run ryml-quickstart in Visual Studio 2015.
Now I'm trying to use the ryml library in my VS 2015 x86 application.
I followed the vcpkg instructions from the "Adding ryml to Visual studio project. #111" conversation. 
The ryml package gets built, and is discoverable in Visual Studio, but when I include 
<ryml/ryml_std.hpp>
<ryml/ryml.hpp>
<c4/ryml_std.hpp>
 , the compiler does not recognize ryml::parse_in_arena().

What am I doing wrong?

**Reply by biojppm** [reply]
@jslezosky please open a different issue to track your new question. For now I will mark your original question as answered.
